src/rt/cover.h

```diff
@@ -106,27 +109,37 @@ typedef enum {
 } cover_dump_t;
 
 typedef enum {
-   COVER_MASK_STMT   = (1 << 0),
-   COVER_MASK_BRANCH = (1 << 1),
-   COVER_MASK_TOGGLE = (1 << 2),
+   COVER_MASK_STMT                        = (1 << 0),
+   COVER_MASK_BRANCH                      = (1 << 1),
+   COVER_MASK_TOGGLE                      = (1 << 2),
+   COVER_MASK_TOGGLE_COUNT_FROM_UNDEFINED = (1 << 8),
+   COVER_MASK_TOGGLE_COUNT_FROM_TO_Z      = (1 << 9),
+   COVER_MASK_TOGGLE_IGNORE_MEMS          = (1 << 10),
+   COVER_MASK_TOGGLE_IGNORE_ARRAYS_FROM   = (1 << 11)
```

Comment:
This isn't really a coverage type though is it? It seems `COVER_MASK_TOGGLE_IGNORE_ARRAYS_FROM` is only added here because of the constraints of the comma separated option parsing interface.

Reply:
Yes, does it mind?

But still the information "exclude / not exclude arrays" still needs to be stored somewhere.